app/angular/index.d.ts

```diff
@@ -0,0 +1,28 @@
+export interface IStorybookStory {
```

Comment:
I think we also have type definitions in the `@types/...` npm organisation. 

Should we move these here? Or add them there also?
Are these even the same type of definitions?

👍

Reply:
I didn't find any typings for apps, only for addons at https://microsoft.github.io/TypeSearch/. 
I think type definitioins and app should be in same package, at least written with typescript like angular. It's not the same typings.